src/main/java/com/northconcepts/datapipeline/foundations/examples/pipeline/StoreDatasetOnDisk.java

```diff
@@ -0,0 +1,40 @@
+package com.northconcepts.datapipeline.foundations.examples.pipeline;
+
+import com.northconcepts.datapipeline.csv.CSVReader;
+import com.northconcepts.datapipeline.foundations.pipeline.Pipeline;
+import com.northconcepts.datapipeline.foundations.pipeline.dataset.Dataset;
+import com.northconcepts.datapipeline.foundations.pipeline.dataset.MvStoreDataset;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+public class StoreDatasetOnDisk {
+    private static final File DATABASE_FOLDER = new File("example/data/output", "mvstore-dataset");
+
+    public static void main(String[] args) throws IOException {
+        Pipeline pipeline = new Pipeline()
+                .setInputAsDataReaderFactory(() -> new CSVReader(new File("example/data/input/credit-balance-01.csv"))
```

Comment:
Let's start using the slightly simpler `setInputAsDataReader()` method going forward -- it does the factory logic behind the scenes.

Reply:
Done